### PR TITLE
Geimaj/approved entities

### PR DESCRIPTION
**Pull request by Geimaj**
implements https://github.com/spinamp/spindexer/issues/48

### CODE REVIEW

File: src/db/migrations/19-add-approved-entities.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
```
Comment (musnit) [inline src/db/migrations/19-add-approved-entities.ts:1]: this migration to add the approved field is coming after the migrations to add all the platforms? so the approved field won't really exist yet in the past migrations?

Comment (Geimaj) [reply src/db/migrations/19-add-approved-entities.ts:1]: ah yeah, will include it int the bootstrap file too

**Comment by musnit** [comment]
@Geimaj looks like the views don't get updated with the new approved fields once the raw tables have them. need to update views too i guess ... hmm seems like it doesn't actually save the view query with *, but rather with individual fields. Can you look into if there is a more general way to solve this?

**Comment by Geimaj** [comment]
@musnit now when we modify tables in migrations we can call `updateViews` which will update the views and their references

**Comment by musnit** [comment]
> @musnit now when we modify tables in migrations we can call `updateViews` which will update the views and their references

cool! can we add something to the readme to make it clear that all schema migrations should call the updateViews method?

**Review by musnit — APPROVED** [review]

**Pull request merged**